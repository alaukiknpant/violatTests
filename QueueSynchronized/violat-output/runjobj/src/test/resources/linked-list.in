{"class": "java.util.LinkedList", "constructor": { "parameters": [] }, "arguments": [], "invocations": [{"method": {"name": "add", "parameters": [{"type": "java.lang.Object"}]}, "arguments": [1]}, {"method": {"name": "add", "parameters": [{"type": "java.lang.Object"}]}, "arguments": [2]}, {"method": {"name": "size", "parameters": []}, "arguments": []}, {"method": {"name": "toArray", "parameters": []}, "arguments": []}, {"method": {"name": "poll", "parameters": []}, "arguments": []}, {"method": {"name": "size", "parameters": []}, "arguments": []}]}
