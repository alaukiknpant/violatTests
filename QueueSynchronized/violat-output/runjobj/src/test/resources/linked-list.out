["true","true","2","[1,2]","1","1"]
